sqlcipher_flutter_libs/lib/sqlcipher_flutter_libs.dart

```diff
@@ -55,3 +55,7 @@ DynamicLibrary openCipherOnAndroid() {
     return DynamicLibrary.open('/data/data/$appId/lib/libsqlcipher.so');
   }
 }
+
+DynamicLibrary openCipherOnMacOS() {
+  return DynamicLibrary.process();
```

Comment:
I wonder if we should move this into the `sqlite3` package.

In the opening code on macOS, I think we could check if `DynamicLibrary.process()` contains sqlite3 symbols (e.g. with `lookup('sqlite3_version').address != 0`). If it does, we use `DynamicLibrary.process()`. If it doesn't, we fallback to the system library.